prepare_package_xml.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/zsh
+zparseopts -A opts -name: -user: -email: -version: -notes:
+if [[ $# -lt 10 ]]
+then
+  echo "Usage ${0} --name NAME, --user USER --email EMAIL --version VERSION --notes NOTES"
+  exit 1
+fi
+NAME="${opts[--name]}"
+USER="${opts[--user]}"
+EMAIL="${opts[--email]}"
+VERSION="${opts[--version]}"
+NOTES="${opts[--notes]}"
+
+TODAY=$(date -u +%Y-%m-%d)
+
+cat package.xml-template_pre \
+    | sed "s/{{{NAME}}}/$NAME/g" \
+    | sed "s/{{{USER}}}/$USER/g" \
+    | sed "s/{{{EMAIL}}}/$EMAIL/g" \
+    | sed "s/{{{TODAY}}}/$TODAY/g" \
+    | sed "s/{{{VERSION}}}/$VERSION/g" \
+    | sed "s/{{{NOTES}}}/$NOTES/g"
+
+source_ext='(c|cc|h|cpp|hpp|m4|w32|ini|frag|cmake|inl|in|py|gnu|yaml|def|pl|S|s|errordata|go|lds|num|asm|mod|peg|mk)'
+doc_ext='(md|json|html|dot|graphml|png|gn|sha1|css|rst|)'
+
+special_docs='(LICENSE*|NOTICE|changelog.txt|CHANGELOG|THIRD-PARTY|README*|readme|METADATA|CONTRIBUTORS|UPDATING|doc.config)'
+special_tests='(ci-test.sh|format-check.sh|run_tests*|sanitizer-blacklist.txt|run-clang-tidy.sh|*_test*.txt|*_vectors.txt|benchmark-build-run.sh|break-tests.sh|generate-coverage.sh|codecov*|test.xml)'
```

Comment:
codecov and *_test*.txt and *_vectors.txt can be stripped

Reply:
removed codecov and skipped.
_test.txt is not that safe to remove because some dependencies use that suffix for "cmake probes".